osbuild

```diff
@@ -17,9 +17,10 @@ if __name__ == "__main__":
     parser = argparse.ArgumentParser(description="Build operating system images")
     parser.add_argument("pipeline_path", metavar="PIPELINE",
                         help="json file containing the pipeline that should be built")
+    parser.add_argument("-c", "--content-store", dest="content_store", metavar="DIRECTORY", type=os.path.abspath,
+                        default=".osbuild/content-store",
```

Comment:
`content-store` rings redundant to my ears. `.osbuild/content`? `.osbuild/objects`? `.osbuild/store`?

Reply:
Should this be `-C`? `make` and `git` (and others?) set a precedent.

Reply:
Good point. I now renamed to `.osbuild/objects`, and skipped the short arg.